src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size()는 중복 원소가 제외된 길이를 반환한다")
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
assertThat(numbers).hasSize(3);
```
참고로, assertj에서는 Collection에 대한 검증 기능도 지원하고 있습니다! :)